stl/inc/format

```diff
@@ -1844,10 +1843,10 @@ template <class _CharT, class _OutputIt, class _Arithmetic>
 _NODISCARD _OutputIt _Fmt_write(_OutputIt _Out, const _Arithmetic _Value) {
     // clang-format on
     // TRANSITION, Reusable buffer
-    array<char, _Format_min_buffer_length> _Buffer;
-    const auto [_End, _Ec] = _STD to_chars(_Buffer.data(), _Buffer.data() + _Buffer.size(), _Value);
+    char _Buffer[_Format_min_buffer_length];
+    const auto [_End, _Ec] = _STD to_chars(_STD begin(_Buffer), _STD end(_Buffer), _Value);

```

Comment:
For the sake of debug performance I would rather avoid calls to `begin` and `end` and simply use the explicit bounds
```suggestion
    const auto [_End, _Ec] = _STD to_chars(_Buffer, _Buffer + _Format_min_buffer_length, _Value);
```